simpa_examples/minimal_optical_simulation.py

```diff
@@ -10,8 +10,8 @@
 import os
 os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
 
-# TODO: Please make sure that a valid path_config.env file is located in your home directory, or that you
-#  point to the correct file in the PathManager().
+# TODO: Please make sure that you have set the correct path to MCX binary and SAVE_PATH in the file path_config.env
+#  located in the simpa_examples directory
```

Comment:
Does the `path_config.env` need to be located in the `simpa_examples` directory? I usually have it in my home directory. Wouldn't any of the options mentioned in the [path management section](https://github.com/IMSY-DKFZ/simpa?tab=readme-ov-file#path-management) of the README work?

Reply:
Rewrote this to mention that the instructions from the README should be followed. The images seem ok :).